imageroot/actions/get-phonebook-credentials/validate-output.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Get phonebook credentials",
+    "$id": "http://nethserver.org/json-schema/task/output/nethvoice/get-phoenbook-credentials",
+    "description": "Get phoenbook credentials",
+    "examples": [
+        { 
+            "host": "10.5.4.1",
+            "port": 3306,
+            "database": "phonebook",
+            "user": "phonebook",
+            "password": "password",
+            "instance_name": "my voice",
+            "nethvoice_host": "voice.nethserver.org"
+        }
+    ],
+    "type": "object"
```

Comment:
The validation needs to check at least if the returned object has all the necessary fields. In this case, I think that we can mark as mandatory all fields in the example.